### DISCUSSION TITLE
Minimal stats embed

### DISCUSSION THREAD

**Discussion by KyleShook** [plausible/analytics#1626] [Anything Else]
I would absolutely love to see the option to have a very minimal version of the embed stats feature. For folks who want to share a little bit but not their sources, locations, pages, etc.
![Screenshot 2022-01-21 221916](https://user-images.githubusercontent.com/32422667/150624487-fb419b41-e5ab-4167-b47b-1cd53963f263.png)

**Comment by metmarkosaric** [comment]
thanks! could be nice! our stats api is better for these types of custom dashboards https://plausible.io/docs/stats-api

**Reply by KyleShook** [reply]
Ah, that makes perfect sense! Thanks for recommending the API!

**Reply by metmarkosaric** [reply]
you're welcome! have fun exploring!
